students/WWhite/lesson07/html_render.py

```diff
@@ -0,0 +1,105 @@
+/>:class Element:
+    indentation = "  "
+    tag_class = ""
+    attributes = ""
+
+    def __init__(self, content=None, **kwargs):
+        if content:
+            self.content = [content]
+            for key, value in kwargs.items():
+                self.attributes += ' {}="{}"'.format(key, value)
```

Comment:
`self.attributes` should not depend on `self.content` since some elements (like `ul`) are instantiated without passing in any content